Engine/Module/ModuleScriptManager.cpp

```diff
@@ -14,25 +14,31 @@
 #include <algorithm>
 
 
-
 bool ModuleScriptManager::Init()
 {
 	APP_LOG_SECTION("************ Module Manager Script ************");
-
+	GetCurrentPath();
 #if GAME
-	gameplay_dll = LoadLibrary(SCRIPT_DLL_FILE);
+	//TODO USE THE NEW FILESYSTEM TO DO THIS
+	bool success = CopyFile(RESOURCE_SCRIPT_DLL_FILE, working_directory.c_str(), false);
```

Comment:
you should change only the RESOURCE_SCRIPT_DLL_FILE for this -> RESOURCES_SCRIPT_DLL_PATH and the bug should be fixed

Reply:
Ok, changing it!